File: SharedProjects/SharedObjects/SharedObjects/Saxotrader/Models/PricePoint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DatasetConstructor.Saxotrader.Models
{
    public class PriceValues
    {
        public int Identifier { get; set; }
        public double Close { get; set; }
        public double High { get; set; }
        public double Interest { get; set; }
        public double Low { get; set; }
        public double Open { get; set; }
        public DateTime Time { get; set; }
        public double Volume { get; set; }
    }
}

File: Forcasting_Models/Database_access/DBModels/PricePoint.py
class PricePoint:
    id = ""
    identifier = ""
    close = ""
    high = ""
    interest = ""
    low = ""
    open = ""
    time = ""
    volume = ""
    def __init__(self, id, identifier, close, high, interest, low, open, time, volume):
        self.id = id
        self.identifier = identifier
        self.close = close
        self.high = high
        self.interest = interest
        self.low = low
        self.open = open
        self.time = time
        self.volume = volume